### PR TITLE
Notification style fixes

### PR DESCRIPTION
**Pull request by ZmagoD**
https://biosistemika.atlassian.net/browse/SCI-531
https://biosistemika.atlassian.net/browse/SCI-533
fix navbar

### CODE REVIEW

File: app/assets/stylesheets/extend/bootstrap.scss [hunk 1]
```diff
@@ -57,3 +57,46 @@
...
+  }
+  .navbar-toggle {
+    display: block;
+    background-color: transparent;
+    background-image: none;
+    border: 1px solid transparent;
+  }
+  .navbar-collapse {
+    border-top: 1px solid transparent;
+    box-shadow: inset 0 1px 0 rgba(255,255,255,0.1);
```
Comment (Ducz0r) [inline app/assets/stylesheets/extend/bootstrap.scss]: Agreed with Hound, try to use one of existing colors, or define a new one in `colors.scss` (and name it using [this nifty tool](http://chir.ag/projects/name-that-color/)).

File: app/assets/stylesheets/extend/bootstrap.scss [hunk 2]
```diff
@@ -57,3 +57,46 @@
...
+  }
+  .navbar-left,.navbar-right {
+    float: none !important;
+  }
+  .navbar-toggle {
+    display: block;
+    background-color: transparent;
+    background-image: none;
+    border: 1px solid transparent;
+  }
```
Comment (Ducz0r) [inline app/assets/stylesheets/extend/bootstrap.scss]: :+1:

**Comment by ZmagoD** [comment]
@Ducz0r I had cleaned up the css a bit but for the rules that is marked as !important the problem is if they are not the rule will not apply "get overridden" and the drop down menu looks  messy

**Comment by Ducz0r** [comment]
Merge away! :+1:

**Pull request merged**